external-crates/move/crates/move-vm-runtime/src/runtime/mod.rs

```diff
@@ -148,17 +144,15 @@ impl MoveRuntime {
         let version_id = pkg.version_id;
         dbg_println!("\n\nPublishing module at {version_id} (=> {original_id})\n\n");
 
-        let data_cache = TransactionDataCache::new(data_cache);
         let link_context = LinkageContext::new(pkg.linkage_table.clone());
 
         // Verify a provided serialized package. This will validate the provided serialized
         // package, including attempting to jit-compile the package and verify linkage with its
         // dependencies in the provided linkage context. This returns the loaded package in the
         // case an `init` function or similar will need to run. This will load the dependencies
         // into the package cache.
-        let pkg_dependencies = package_resolution::resolve_packages(
+        let pkg_dependencies = PackageStore::new(data_cache).resolve_packages(
```

Comment:
The comment or this overall function is probably wrong now that the data cache is gone.